### ISSUE TITLE
[Bug]: Still showing as playing Dota 2 on Discord

### ISSUE THREAD

**Issue by PriscillAAH** [andsfonseca/dota2-rpc-client#7]
### Are you on the latest version?

- [X] I\'m on the latest version

### Is there a fix in progress related to your proposal?

- [X] I have searched the existing issues

### Describe the bug

Followed steps of installation still just shows me as Playing Dota 2. Tried manually putting cfg file in cfg folder still doesn't work. 

### Dota 2 Patch

7.31d

**Comment by andsfonseca** [comment]
@PriscillAAH Can you perform some tasks for me?

* First uninstall the service if you have it installed.
* Open the terminal in the application folder and enter the following command:
```
dota2-rpc-client -f
```
* Send me the output of the program

**Comment by PriscillAAH** [comment]
Yep I already installed again and that's when it started working. Here's a screenshot of it working:

![Discord_9skOkSYZW0](https://user-images.githubusercontent.com/32128443/173097305-6148f918-22cf-4065-be6f-d88bed4a64c5.png)
